regression/smt2_solver/function-applications/function-application2.desc

```diff
@@ -0,0 +1,7 @@
+CORE
+function-application2.smt2
+
+^EXIT=0$
+^SIGNAL=0$
+^line 5: type mismatch in function definition: expected `\(_ BitVec 16\)' but got `\(_ BitVec 8\)'$
```

Comment:
So the exit code is useless?

Reply:
Most solvers seem to do the "continued-execution" error handling from the SMT-LIB standard; yes, the exit code is then used for SAT/UNSAT only.